notdeft.el

```diff
@@ -2378,7 +2378,8 @@ If STR is nil, clear the filter."
 In particular, update `notdeft-current-files'.
 Get the character from the variable `last-command-event'."
   (interactive)
-  (let ((char last-command-event))
+  (let ((char (let ((buffer-read-only nil))
+                (car (funcall input-method-function last-command-event)))))
```

Comment:
Thanks for the PR. The documentation seems to suggest that input-method-function could be nil. Can that be the case here, and should we check for that?

Reply:
Yeah, that's an interesting question. Surely, you are right and it is better to always check such things. On my system, `input-method-function` is either simple `list` or `quile-input-method`. If the variable is forced to `nil`, then almost everything works as usual, except for my hack (because it's a hack:). The documentation only recommends setting its value to `nil` when implementing input methods using `read-event` or `read-key-sequence` to avoid recursion. And it is not recommended to set it directly, but no one will give a guarantee.
I think I'll add a check ...